Fixed some details in with-office-365.md

dariomws opened the pull request:
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6258.

scanum commented:
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

JohanFreelancer9 commented on Teams/rooms/with-office-365.md:
```diff
@@ -150,22 +149,25 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
+   
    For detailed instructions, see [Assign licenses to user accounts with Office 365 PowerShell](https://docs.microsoft.com/office365/enterprise/powershell/assign-licenses-to-user-accounts-with-office-365-powershell#use-the-microsoft-azure-active-directory-module-for-windows-powershell).
+   
+   You can also add Phone System capabilities to this account, but previously you have to configure it. See [What is Phone System?](../what-is-phone-system-in-office-365.md) for more details. This example adds the PSTN Domestic and International Calling Plan.
```
```suggestion
   You can also add Phone System capabilities to this account, but you have to configure it first. See [What is Phone System?](../what-is-phone-system-in-office-365.md) for more details. This example adds the PSTN Domestic and International Calling Plan:
```

JohanFreelancer9 commented on Teams/rooms/with-office-365.md:
````diff
@@ -150,22 +149,25 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
    ```  -->
 
-   Next, you can add a license using the `Set-MsolUserLicense` <!--Set-AzureADUserLicense --> cmdlet. In this case, $strLicense is the SKU code that you see (for example, contoso:STANDARDPACK).
+   Next, you can add a license using the `Set-MsolUserLicense` <!--Set-AzureADUserLicense --> cmdlet. This example adds the Meeting Room license to this account:
````
```suggestion
   Next, you can add a license using the `Set-MsolUserLicense` <!--Set-AzureADUserLicense --> cmdlet. This example adds the Meeting Room license to the account:
```

JohanFreelancer9 reviewed (changes requested):
Copy review - minor suggestions.

scanum commented:
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

cichur commented:
Hi @dariomws ,
Were you able to test the changes that you made to the article? 
Cindy

dariomws commented:
Hi @cichur,

I didn't test them as all the changes are to simplify the content. I didn't rewrite the code or commands.

- $strLicense variable is the only fix that I applied. This variable is called in the Set-MsolUserLicense but never used. I think the writer forgot to add something in the third line:
![image](https://user-images.githubusercontent.com/33589238/101544282-2b4f0f80-3984-11eb-931c-66381b3bacd0.png)
- I removed the $acctUpn variable and now the cmdlets pass the UPN value directly. In this way, we have only one line of code.
- I removed the $strLicense variable and then passed a license example value directly. I think MEETING_ROOM is the best example here, as it should usually be.
- I removed the last section with an example as it was no more than all the commands together. Each step use examples in the "Add a device account" title.
- I also added a comment/note about Phone System capabilities that was in the example section and not in the "Add a device account" title. Also updated wording for this comment as it was updated.

Please let me know if you need more information.

cichur commented:
HI @dariomws and @JohanFreelancer9 
 I'm not familiar with this content and am taking over for Lana, whose contract ended. :-(
I'm looking at the changes in line 199. It looks like an entire note was removed from the article. I just want to  make sure everything is correct before I merge the changes. :-)
Cindy

dariomws commented:
Hi @cichur, don't worry, ask all the questions you need.

I didn't remove that note, it's there in line 199 of the updated article:
![image](https://user-images.githubusercontent.com/33589238/101623595-6e969600-39f7-11eb-97ac-70b927cfc3d0.png)

cichur commented:
Hi @dariomws,
Thank you. I do see it now. I'll go ahead and merge the PR. :-)
Thank you very much, Cindy
